validators/cli_validator.py

```diff
@@ -0,0 +1,18 @@
+from validators import BaseValidator
+import tempfile
+import os
+
+class CliValidator(BaseValidator):
+    
+    def is_valid(self, content:str) -> bool:
+        
+        fd, abspath = tempfile.mkstemp(suffix=".yaml")
+        with open(abspath, 'w+') as f:
+            f.write(content)
+
+        try:
+            ex = os.system('circleci config validate %s' % abspath)
```

Comment:
os.system is good, but the new preferred way to do this in python is with the subprocess library. Take a look at this below, and try out subprocess.run. It's a good learning experience.

https://docs.python.org/3/library/subprocess.html

Reply:
I changed it, and it shows me the same error as before, so I hope it works ;)